sections/security/secretmanagement.md

```diff
@@ -0,0 +1,31 @@
+# Extract secrets from config files or use NPM package that encrypts them
+
+### One Paragraph Explainer
+The most common and secure way to provide a Node.js application access to keys and secrets is to store them using environment variables on the system where it is being run. Once set, these can be accessed from the global `process.env` object. 
```

Comment:
I think you should start with the concept of vaults - this what makes the real difference. Most developers do *finally put the code in env var, only they first store them in .env files within the source control... The critical part is to fetch the secrets from the vault during deployment, where only the deployment component has access to the vault, and it inject the secret into the production env var @js-kyle